app/frontend/src/features/donations/DonationsBox.tsx

```diff
@@ -0,0 +1,203 @@
+import { Divider, makeStyles, Typography } from "@material-ui/core";
+import Button from "components/Button";
+import {
+  DONATIONSBOX_CURRENCY,
+  DONATIONSBOX_MONTHLY,
+  DONATIONSBOX_NEXT,
+  DONATIONSBOX_ONETIME,
+  DONATIONSBOX_TEXT,
+  DONATIONSBOX_TITLE,
+  DONATIONSBOX_VALUES,
+} from "features/donations/constants";
+
+const useStyles = makeStyles((theme) => ({
+  donationsBox: {
+    padding: theme.spacing(2),
+    border: `2px solid ${theme.palette.grey[200]}`,
+    borderRadius: theme.shape.borderRadius * 2,
+  },
+
+  donationsBoxRow: {
+    gridTemplateColumns: "repeat( auto-fit, minmax(72px, 1fr) )",
+    gridTemplateRows: "2.75rem",
+    display: "grid",
+    gridGap: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+  },
+
+  buttonSecondary: {
+    border: `2px solid ${theme.palette.grey[200]}`,
+    backgroundColor: theme.palette.grey[200],
+    "&&": {
+      borderRadius: "0.5rem",
+      boxShadow: "initial",
+    },
+    "&:hover": {
+      border: `2px solid ${theme.palette.primary.main}`,
+      backgroundColor: theme.palette.background.paper,
+    },
+    "&:hover > .MuiButton-label": {
+      color: theme.palette.primary.main,
+      transition: "color 250ms cubic-bezier(0.4, 0, 0.2, 1)",
```

Comment:
try to use [the theme](https://next.material-ui.com/customization/transitions/#durations):
```suggestion
      transition: `color ${theme.transitions.duration.short} ${theme.transitions.easing.easeInOut}`,
```